src/Microsoft.Graph/Requests/Generated/TeamArchiveRequestBuilder.cs

```diff
@@ -27,10 +27,10 @@ public partial class TeamArchiveRequestBuilder : BaseActionMethodRequestBuilder<
         public TeamArchiveRequestBuilder(
             string requestUrl,
             IBaseClient client,
-            bool? shouldSetSpoSiteReadOnlyForMembers)
+            bool shouldSetSpoSiteReadOnlyForMembers)
             : base(requestUrl, client)
         {
-            this.SetParameter("shouldSetSpoSiteReadOnlyForMembers", shouldSetSpoSiteReadOnlyForMembers, true);
```

Comment:
And this.

Reply:
https://github.com/microsoftgraph/msgraph-metadata/commit/cf293dcc71b9336c294418ea609dc1b4e5d599a5#diff-e34459d76b3a635d414d3549ff656c4cL10530